.travis.yml

```diff
@@ -2,16 +2,14 @@ services:
 - docker
 script:
 - docker build --tag todo-app-dev --target development .
-- docker build --tag todo-app-test --target test .
-- docker build --target production --tag anniebedford/todo-app-prod . 
-#docker push registry.heroku.com/devops-dotnet/web
+- docker build --tag todo-app-test --target test . 
+after_script:
```

Comment:
Is there a reason for introducing this? Note that this means your tests now run after you deploy, which means we're higher risk of deploying broken code